server/sonar-server/src/main/java/org/sonar/server/user/UserSessionFilter.java

```diff
@@ -52,18 +53,31 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     HttpServletResponse response = (HttpServletResponse) servletResponse;
 
     ThreadLocalSettings settings = platform.getContainer().getComponentByType(ThreadLocalSettings.class);
+    DefaultOrganizationCache defaultOrganizationCache = platform.getContainer().getComponentByType(DefaultOrganizationCache.class);
     UserSessionInitializer userSessionInitializer = platform.getContainer().getComponentByType(UserSessionInitializer.class);
 
-    settings.load();
+    defaultOrganizationCache.load();
+    try {
+      settings.load();
+      try {
+        doFilter(request, response, chain, userSessionInitializer);
+      } finally {
+        settings.unload();
+      }
+    } finally {
+      defaultOrganizationCache.unload();
+    }
+  }
+
+  private void doFilter(HttpServletRequest request, HttpServletResponse response, FilterChain chain, @Nullable UserSessionInitializer userSessionInitializer) throws IOException, ServletException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make "doFilter" a "static" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2325)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 196 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS00103)